client/thin-replica-client/src/thin_replica_client.cpp

```diff
@@ -723,18 +723,24 @@ void ThinReplicaClient::pushUpdateToUpdateQueue(std::unique_ptr<EventVariant> up
 
   // push update to the update queue for consumption by the application using
   // TRC
+  const auto& block_id = std::get<Update>(*update).block_id;
+  auto num_events = std::get<Update>(*update).kv_pairs.size();
```

Comment:
The `update` will be of type `EventGroup` when event groups are enabled, there is no concept of block_id and kv_pairs in type EventGroup.